tests/integration/config.yml

```diff
@@ -0,0 +1,11 @@
+# config.yml for integration test of exodus lambda function
```

Comment:
I don't think this should require a config file checked into the repo... or at least, it should not be the only way to configure the tests.  It's important to be able to run the same test suite against different environments (e.g. production, a local dev env, or staging env after a deployment was put there).

Also, these tests shouldn't be run by default in CI or when someone runs "pytest" or "tox" locally.  They probably are, at the moment.

I think these two things are related.  Here's a suggestion to fix both issues:

1. Use pytest_addoption feature to add a new option to pytest command which accepts the base URL of the CDN.

2. Set up integration tests so that they skip unless that option is provided.

Overall that would look something like this - in conftest.py (not tested):

```python
def pytest_addoption(parser):
    parser.addoption("--cdn-test-url", action="store", help="enable integration tests against this CDN")

@pytest.fixture
def cdn_test_url(request):
    out = request.config.getoption("--cdn-test-url")
    if not out:
      pytest.skip("Test skipped without --cdn-test-url")
    return out
```

Then each integration test, if it accepts cdn_test_url as an argument, will be skipped if --cdn-test-url is not provided.